src/librepgp/stream-def.h

```diff
@@ -54,7 +54,7 @@
 #define ST_FROM ("From")
 
 /* Preallocated cache length for AEAD encryption/decryption */
-#define PGP_AEAD_CACHE_LEN (PGP_INPUT_CACHE_SIZE + PGP_AEAD_MAX_TAG_LEN)
+#define PGP_AEAD_CACHE_LEN (PGP_INPUT_CACHE_SIZE + 2 * PGP_AEAD_MAX_TAG_LEN)
```

Comment:
Just curious, why 2?

Reply:
Found my answer in stream-write.cpp :)

Reply:
Yeah, this is to include the tag for the last block + the final tag after the encrypted stream.